.travis.yml

```diff
@@ -62,6 +62,12 @@ before_deploy:
 - echo "Deploying $RELEASE_PKG_FILE to GitHub releases."
 - export GIT_TAG="latest"
 - export TAG=false
+- if [ ! -z "$TRAVIS_TAG" ] ; then
```

Comment:
: If the current build is for a git tag, this variable is set to the tag’s name.